docker/api

```diff
@@ -0,0 +1,11 @@
+FROM ruby
+MAINTAINER Jonathan Claudius
+ENV PROJECT=github.com/mozilla/ssh_scan_api
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
+RUN gem install bundler
+RUN bundle install
+CMD /app/bin/ssh_scan_api
```

Comment:
Consider leaving CMD or ENTRYPOINT out of the Dockerfile and make explicit in docker-compose.yml.  This is friendlier to other containerizers.